.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -345,7 +345,25 @@ Minikube is the only caveat - it's not reachable publicly so don't run a proxy t
   * [ ] Verify that clicking on a rows connect button renders a dialogue on manual instructions with `Step 2` login value matching the rows `name` column
   * [ ] Verify searching for `name` or `labels` in the search bar works
   * [ ] Verify you can sort by `name` colum
-* [ ] Test Kubernetes exec via WebSockets - [client](https://github.com/kubernetes-client/javascript/blob/45b68c98e62b6cc4152189b9fd4a27ad32781bc4/examples/typescript/exec/exec-example.ts)
+
+### Kubernetes exec via WebSockets/SPDY
+
+To control usage of websockets on kubectl side environment variable `KUBECTL_REMOTE_COMMAND_WEBSOCKETS` can be used:
```

Comment:
Include test for v4 websocket protocol used by other go clients

Reply:
I've returned previous check: https://github.com/gravitational/teleport/pull/41683/commits/2a1d36f5c7835a33703b5ecb2b455f77d6d1dbce